src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -42,13 +42,12 @@
 
 # estimated execution time (in minutes) for each tool to process a chunk of data
 TASK_META = {'checkm2': {'chunk_size': 5000, 'exe_time': 60},
-             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 90}}
-NODE_TIME_LIMIT = 10  # hours
+             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65}}
+NODE_TIME_LIMIT = 5  # hours  # TODO: automatically calculate this based on tool execution time and NODE_THREADS
 MAX_NODE_NUM = 100  # maximum number of nodes to use
 # The THREADS variable controls the number of parallel tasks per node
-# we want to set it to 1 (execute tasks one by one) because batch parallelization is handled by the
-# compute_genome_attribs.py script
-NODE_THREADS = 1
+# TODO: make this configurable based on tool used
+NODE_THREADS = 4
```

Comment:
Can you document the reason for this value of NODE_THREADS

Reply:
👍